docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -193,15 +193,29 @@ The following error message codes may appear on ZSS log. Use the following messa
 
 ### ZWES1601I
 
-  Server is ready to accept JWT `with`(or `without`) fallback to legacy tokens
+ZSS server is ready to accept JWT with (or without) fallback to legacy tokens.
 
-  **Reason:**
+**Reason:**
 
-  Server is ready to accept JWT `with` or `without` fallback to legacy tokens.
+This message indicates that the ZSS (Zowe Security Server) has successfully started and is ready to accept JWT (JSON Web Token)-based authentication. Depending on your configuration, it may also accept legacy Zowe tokens as a fallback.
 
-  **Action:**
+**What does "with (or without) fallback to legacy tokens" mean?**
+
+ZSS can support two types of authentication tokens:
+- **JWT (JSON Web Tokens)** – the default and more secure method.
+- **Legacy tokens** – used in older versions of Zowe.
```

Comment:
"Legacy tokens" here means that something can log into ZSS with the non-JWT types of logins ( /login URL, basic auth, cert auth, etc)